quint/src/itf.ts

```diff
@@ -126,6 +173,60 @@ export function toItf(vars: string[], states: QuintEx[]): Either<string, ItfTrac
   })
 }
 
-// export function ofItf(itf: any[]): Either<string, QuintEx[]> {
-//   const stateToExpr = (any: )
-// }
+export function ofItf(itf: ItfTrace): QuintEx[] {
```

Comment:
I'm far from being a specialist in English propositions, but perhaps `fromItf` makes more sense here

Reply:
I think either can work grammatically. I don't have a strong preference here, but assuming you don't either from the approval and tone of comment :)

If I'm wrong and you end up finding this confusing, let me know and i'll fix it in a followup!